free-programming-books.md

```diff
@@ -1029,7 +1028,7 @@ Original Source: [List of freely available programming books](http://web.archive
 * [Developer Workbooks](http://developer.force.com/workbook)
 * [Force.com Fundamentals](https://developer.salesforce.com/page/Force_Platform_Fundamentals)
 * [Force.com Platform Fundamentals: An Introduction to Custom Application Development in the Cloud](http://www.lulu.com/shop/salesforcecom/forcecom-platform-fundamentals/ebook/product-17381451.html)
-* [Heroku Postgres](http://media.developerforce.com/workbooks/HerokuPostgres_Workbooks_Web_Final.pdf) (PDF)
+* [Heroku Postgres](http://www.scribd.com/doc/256342154/herokupostgres-workbooks-web-final-13-pdf) (PDF)
```

Comment:
I didn't know issuu either. I think we should adopt the same policy for all the "uploading" platforms. Issuu, scribd, gdrive, dropbox, etc.

Some of the files uploaded on these platforms and shared here are certainly legitimate, but their legitimacy is much harder to check without having an account on all these platforms. Also, I don't see why an author or a copyright holder would decide to exclusively share on issuu or scribd, knowing that the platform will most probably monetize the content in some way.

Same for drive and dropbox without the monetization issue. Anyone could upload anything there.